docs/introduction.rst

```diff
@@ -35,40 +35,104 @@ Analysis steps
 
 R1 to DL1
 ---------
-Usage of
 
-``lstchain.scripts.lstchain_data_r0_to_dl1``
+MC data
+^^^^^^^
```

Comment:
Not sure if this sub-section will be easily understood on Sphinx. Maybe just writing in a single section on both MC and real data, or listing them, might be more useful. There might be better ways to make it easily readable on Sphinx.

Maybe referencing the `lst_analysis_workflow` would be helpful to understand the sub-sections where there are distinct procedures for MC and real data, and when they are similar.

This is also applicable to the `DL2 to DL3` section

Reply:
<img width="481" alt="Capture d’écran 2023-05-04 à 11 57 53" src="https://user-images.githubusercontent.com/4263646/236173279-aea4dbb2-5a14-4512-abbd-f71d0f305747.png">

Reply:
It looks ok to me this way, no?
I added a sentence with link to the workflows.

Reply:
Yes, it looks ok!